### PR TITLE
Add books-table reset-functionality

### PR DESCRIPTION
**Pull request by Tartsi**

### CODE REVIEW

**Comment by Tartsi** [comment]
Do not merge yet, I will fix test coverage!

**Comment by Tartsi** [comment]
Mergeable!

File: src/tests/test_dataprocessing.py [hunk 1]
```diff
@@ -19,4 +20,26 @@ def test_fail_to_add_book(self):
 
     def test_get_all_books(self):
         with app.app_context():
-            self.assertGreater(len(dataprocessing.get_all_books()), 0)
+            self.assertEqual(len(dataprocessing.get_all_books()), 0)
```
Comment (Wincewind) [inline src/tests/test_dataprocessing.py:23]: Why does this make the tests pass? I don't see any changes with the actions workflow so shouldn't this still return the 2 rows added with test schema?

Comment (Wincewind) [reply src/tests/test_dataprocessing.py:23]: Otherwise looks good!

**Review by Wincewind — APPROVED** [review]
Sorry for the wait, great work1

**Pull request merged**